mithril-common/src/digesters/cardano_immutable_digester.rs

```diff
@@ -502,7 +655,48 @@ mod tests {
     }
 
     #[tokio::test]
-    async fn cache_read_failure_dont_block_computation() {
+    async fn merkle_tree_computation_is_quicker_with_a_full_cache() {
```

Comment:
This test is probably not necessary (very redundant) and not that relevant after all :thinking:

Reply:
I initially thought it was a good way to demonstrate the behavior for each distinct process from start to finish, even if the cache reading process is shared.
I understand and I agree, the test is redundant, so I will remove it 👍

Reply:
Done ✅